### PR TITLE
port todo example to otp

### PR DESCRIPTION
**Pull request by alsuren**
This is the example that we will be using in our demo.

I can make a second PR once https://github.com/wasmCloud/interfaces/pull/10 is merged and released, or we can leave this PR open until then (this won't block anything on our side).

### CODE REVIEW

File: actor/todo/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+use anyhow::{anyhow, Result};
+use log::{info, trace, warn};
+use serde::{Deserialize, Serialize};
+use wasmbus_rpc::actor::prelude::*;
+use wasmcloud_interface_httpserver::{HttpRequest, HttpResponse, HttpServer, HttpServerReceiver};
+use wasmcloud_interface_keyvalue::{
+    IncrementRequest, KeyValue, KeyValueSender, SetAddRequest, SetDelRequest, SetRequest,
+};
+
+// FIXME: contribute this upstream
```
Comment (alsuren) [inline actor/todo/src/lib.rs:12]: I can make a second PR once https://github.com/wasmCloud/interfaces/pull/10 is merged and released, or we can leave this PR open until then (this won't block anything on our side).

**Comment by autodidaddict** [comment]
Interfaces PR is merged. @brooksmtownsend can we publish to crates?

**Comment by brooksmtownsend** [comment]
I'm just getting to the CI now, but should be able to start on interfaces this AM. I'll CC you all on the actions PR once it's ready

**Comment by brooksmtownsend** [comment]
PR https://github.com/wasmCloud/interfaces/pull/12

**Comment by stevelr** [comment]
I updated this to use the latest httpserver interface library, removing the need for the Shim. and put the results in PR: https://github.com/wasmCloud/examples/pull/70. (Haven't tested it)

**Review by autodidaddict — APPROVED** [review]

**Pull request merged**